check_if_image_instrumentable.sh

```diff
@@ -86,9 +86,12 @@ fi
 
-supported_centos_libc=(glibc-2.26-33.amzn2.0.1.x86_64.rpm* glibc-2.26-33.amzn2.0.2.x86_64.rpm* glibc-2.17-307.el7.1.x86_64.rpm glibc-2.28-72.el8.x86_64.rpm glibc-2.17-106.el7.x86_64.rpm glibc-2.26-32.amzn2.0.2.x86_64.rpm glibc-2.26-32.amzn2.0.1.x86_64.rpm glibc-2.17-292.el7.x86_64.rpm glibc-2.17-260.el7_6.6.x86_64.rpm glibc-2.17-222.el7.x86_64.rpm glibc-2.17-260.el7.x86_64.rpm glibc-2.17-260.el7_6.3.x86_64.rpm glibc-2.17-196.el7_4.2.x86_64.rpm glibc-2.17-196.el7.x86_64.rpm glibc-2.17-157.el7_3.1.x86_64.rpm glibc-2.17-106.el7_2.8.x86_64.rpm)
+supported_centos_libc=(glibc-2.26-33.amzn2.0.1.x86_64.rpm* glibc-2.26-33.amzn2.0.2.x86_64.rpm* glibc-2.17-307.el7.1.x86_64.rpm glibc-2.28-72.el8.x86_64.rpm glibc-2.17-106.el7.x86_64.rpm glibc-2.26-32.amzn2.0.2.x86_64.rpm glibc-2.26-32.amzn2.0.1.x86_64.rpm glibc-2.17-292.el7.x86_64.rpm glibc-2.17-260.el7_6.6.x86_64.rpm glibc-2.17-222.el7.x86_64.rpm glibc-2.17-260.el7.x86_64.rpm glibc-2.17-260.el7_6.3.x86_64.rpm glibc-2.17-196.el7_4.2.x86_64.rpm glibc-2.17-196.el7.x86_64.rpm glibc-2.17-157.el7_3.1.x86_64.rpm glibc-2.17-106.el7_2.8.x86_64.rpm 
+                       glibc-2.17-106LI.el7.centos.x86_64.rpm glibc-2.17-107LI.el7.centos.8.x86_64.rpm glibc-2.17-158LI.el7.centos.1.x86_64.rpm glibc-2.17-197LI.el7.centos.2.x86_64.rpm glibc-2.17-197LI.el7.centos.x86_64.rpm glibc-2.17-223LI.el7.centos.x86_64.rpm glibc-2.17-261LI.el7.centos.3.x86_64.rpm glibc-2.17-261LI.el7.centos.6.x86_64.rpm glibc-2.17-261LI.el7.centos.x86_64.rpm glibc-2.17-293LI.el7.centos.x86_64.rpm glibc-2.17-308QCRS.el7.centos.1.x86_64.rpm glibc-2.17-318QCRS.el7.centos.x86_64.rpm glibc-2.17-324QCRS.el7.centos.x86_64.rpm glibc-2.28-152QCRS.el8.centos.x86_64.rpm glibc-2.28-73QCRS.el8.centos.x86_64.rpm glibc-common-2.17-106LI.el7.centos.x86_64.rpm glibc-common-2.17-107LI.el7.centos.8.x86_64.rpm glibc-common-2.17-158LI.el7.centos.1.x86_64.rpm glibc-common-2.17-197LI.el7.centos.2.x86_64.rpm glibc-common-2.17-197LI.el7.centos.x86_64.rpm glibc-common-2.17-223LI.el7.centos.x86_64.rpm glibc-common-2.17-261LI.el7.centos.3.x86_64.rpm glibc-common-2.17-261LI.el7.centos.6.x86_64.rpm glibc-common-2.17-261LI.el7.centos.x86_64.rpm glibc-common-2.17-293LI.el7.centos.x86_64.rpm glibc-common-2.17-308QCRS.el7.centos.1.x86_64.rpm glibc-common-2.17-318QCRS.el7.centos.x86_64.rpm glibc-common-2.17-324QCRS.el7.centos.x86_64.rpm glibc-common-2.26-33.amzn2.0.1.x86_64.rpm glibc-common-2.26-33.amzn2.0.2.x86_64.rpm glibc-common-2.28-152QCRS.el8.centos.x86_64.rpm glibc-common-2.28-73QCRS.el8.centos.x86_64.rpm glibc-minimal-langpack-2.28-152QCRS.el8.centos.x86_64.rpm glibc-minimal-langpack-2.28-73QCRS.el8.centos.x86_64.rpm)
```

Comment:
entries having `common `and `minimal `are not required.

Reply:
Done!